Script/char.dat

```diff
@@ -2523,10 +2817,11 @@ hunter
   HostileReplies == "\"Your head will look fine above my fireplace!\"";
   FriendlyReplies =
   {
-    5,
+    6,
     "\"A man is not a man unless he has lost his left arm in a battle against a polar bear.\"",
     "\"Bears, ogres, slaves, farmers... Ah, there's so much to hunt here!\"",
     "\"I am the Great White Hunter. Get out of My way!\"",
+    "\"Oh, I only hunt for the sport. I'm vegetarian.\"",
```

Comment:
I liked the joke because it was suggested in the original devs' plans (see Doc/Data/Dialog.txt). But I'll remove the line.